README.md

```diff
@@ -21,15 +21,15 @@ DBI::dbDisconnect(connection)
 In this example we just extract username from url `username` parameter.
 ```
 get_user <- function(session) {
-  parseQueryString(session$clientData$url_search)$username
+  isolate(parseQueryString(session$clientData$url_search)$username)
```

Comment:
This looking weird.
If there is really a problem with reactivity I suggest to use.
```suggestion
  parseQueryString(isolate(session$clientData$url_search))$username
```

Reply:
@galachad `session$clientData$url_search` is reactive value we want to source just once per session. As this is called outside reactive context it's enough to isolate it.